keylime-ansible-gcp/playbook.yml

```diff
@@ -0,0 +1,22 @@
+---
+- hosts: all
+  become: true
+  become_user: root
+  pre_tasks:
+    - name: Probe python installation
+      raw: command -v python3 || true
+      changed_when: false
+      register: python_available
+    - name: Bootstrap a host without python installed
+      raw: dnf install python3 python3-libselinux
+      when: "'python' not in python_available.stdout"
+    - name: Ensure Python 3 is available
+      dnf:
+        name: python3
+        state: present
+    - name: Install libselinux-python3
+      dnf:
+        name: libselinux-python3
+        state: present
+  roles:
+    - keylime-gcp
```

Comment:
Can you add a new line at the end of the files? It's a pretty common thing to require and most code editors have some setting to enable it if it's not the default.